docs/advanced-topics/custom-serialization.md

```diff
@@ -3,7 +3,7 @@ id: custom-serialization
 title: Custom Serialization
 ---
 
-When using `RPC`'s, `NetworkVariable`'s or any other Netcode for GameObjects (Netcode) related task that requires serialization. The Netcode uses a default serialization pipeline that looks like this:
+When using `RPC`s, `NetworkVariable`'s or any other Netcode for GameObjects (Netcode) related task that requires serialization. The Netcode uses a default serialization pipeline that looks like this:
```

Comment:
```suggestion
When using `RPCs, `NetworkVariable`'s or any other Netcode for GameObjects (Netcode) related task that requires serialization, the Netcode uses a default serialization pipeline that looks like this:
```

Reply:
I made a commit with the other change and also removed the errant apostrophe from `NetworkVariable`'s.

(Just want to note that the only thing I changed about this line to begin with was removing an errant apostrophe from `RPC`'s. The rest of this copy predated this PR, but it it a valid grammatical change so I went ahead and made it.)